serverlessworkflow/modules/ROOT/pages/cloud/operator/enabling-jobs-service.adoc

```diff
@@ -0,0 +1,170 @@
+= Managing Jobs Service with the Operator
+:compat-mode!:
+// Metadata:
+:description: Configure Jobs Service using the `SonataFlowPlatform` CR.
+:keywords: sonataflow, serverless, operator, kubernetes, jobs service
+
+
+This document describes how to configure the Jobs Service instance usimg the SonataFlowPlarform CR.
+
+== Automate the Jobs Service instance management with the `SonataFlow` Operator
+
+While it is possible to deploy the Jobs Service manually, the Operator also provides a convenient way to combine it with the
+namespace configuration via the `SonataFlowPlatform` CR. With this approach, the Operator will take care of configuring the
+Jobs Service instance and ensure it is in sync with the specification in the CR. When the operator manages the Jobs Service lifecycle,
+it will inject properties in `SonataFlow` workflows at creation time to enable the workflows to communicate with the Jobs Service
+instance during their execution, removing the need to add these properties as part of the `SonataFlow` workflow CR instance.
+
+== Configuring Jobs Service in the SonataFlowPlatformCR
+
+To enable the deployment of a Jobs Service instance, the `SonataFlowPlatform` CRD exposes a set of fields that allow the user to 
+configure the running instance. 
+
+==== Ephemeral persistence
+The basic runtime is to deploy the Jobs Service with an ephemeral backend running in the same container
+as the Jobs Service runtime.
+
+[source,yaml,subs="attributes+"]
+---
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  services:
+    jobService: {}
+---
+
+When executing this manifest, the operator will reconcile generating a pod hosting the Jobs Service:
+
+[source,shell,subs="attributes+"]
+---
+$>kubectl get pod -n sonataflow
+NAME                                               READY   STATUS    RESTARTS   AGE
+sonataflow-platform-jobs-service-cdf85d969-sbwkj   1/1     Running   0          108s
+---
+
+Keep in mind that this setup is not recommended for production environments, chiefly because the data is not persisted when the pod restarts.
```

Comment:
```suggestion
Keep in mind that this setup is not recommended for production environments, especially because the data does not persist when the pod restarts.
```